<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>caloricTableWindow</class>
 <widget class="QDialog" name="caloricTableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">setWindowTitle:&quot;CalorieTable&quot;;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_caloricTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="lineEdit_foodItemSearch">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_foodItem">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>371</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_changeAddFoodItemPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>371</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #008080;
color: white;</string>
     </property>
     <property name="text">
      <string>Додати власний продукт</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_addFoodItem">
    <widget class="QPushButton" name="pushButton_addFoodItemToTable">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>380</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #008080;
color: white;</string>
     </property>
     <property name="text">
      <string>Додати продукт в таблицю</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_enterProductName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Введіть назву продукту</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_foodItemName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_caloricContent">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_proteinContent">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_enterCalorie">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Введіть вміст калорій(на 100г)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ifKnow">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(Якщо відомо)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_enterProtein">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Введіть вміст білків(на 100г)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_enterCarbohydrate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Введіть вміст вуглеводів(на 100г)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_carbohydrateContent">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_enterFat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Введіть вміст жирів(на 100г)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_fatContent">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancelAddProduct">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #008080;
color: white;</string>
     </property>
     <property name="text">
      <string>Скачувати</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
